label_studio/tests/conftest.py

```diff
@@ -769,3 +770,13 @@ def tick_clock(_, seconds: int = 1) -> None:
     now += timedelta(seconds=seconds)
     freezer = freeze_time(now)
     freezer.start()
+
+
+@pytest.fixture(autouse=True)
+def _raise_nplusone(request):
```

Comment:
Why is this called `_raise_nplusone`? I do see that there's an nplusone setting NPLUSONE_RAISE that configures whether nplusone will raise an exception when it detects an n+1 query: https://github.com/jmcarp/nplusone#customizing-notifications

Do tests start failing if that setting is enabled?

Reply:
`_raise_nplusone` raises nplusone issues unless they are marked to skip by `skip_nplusone`

Reply:
Another potential concern: https://github.com/jmcarp/nplusone#generic says that when using the Profiler one has to import the relevant extension for one's ORM. In this case I think we'd also be okay if we had NPlusOneMiddleware added to middleware during CI, but for what it's worth, I don't think DEBUG=True on CI by default.

My major underlying concern is that there aren't that many tests marked `skip_nplusone`, I would have expected many more failures.

Reply:
Interesting that it didn't fail without the extension, and it seems to detecting nplusone issues.  Its possible that when adding the extension it will catch more issues.  Wish the docs specified the reason for it.  Anyway, I'll test with the extension and see if we gain any added benefits from it.